action.yml

```diff
@@ -180,17 +237,8 @@ runs:
         echo "step_summary_file=${STEP_SUMMARY_FILE}" >> $GITHUB_OUTPUT
         echo "issue_file=${ISSUE_SUMMARY_FILE}" >> $GITHUB_OUTPUT
 
-    - name: Cache .terraform
-      id: cache
-      uses: actions/cache@v4
-      if: ${{ fromJson(steps.component.outputs.settings).enabled }}
-      with:
-        path: |
-          ./${{ steps.vars.outputs.component_path }}/.terraform
-        key: ${{ steps.vars.outputs.cache-key }}
-
```

Comment:
Pls, revert the cache step. 
It speeds up the action a lot  by caching terraform providers per component

Reply:
done